### PR TITLE
Make if name == 'main' construction for start app

### PR DESCRIPTION
**Pull request by no-0-name**
make if __name__ == '__main__' construction for start app

### CODE REVIEW

**Comment by w3bdesign** [comment]
@no-0-name 

Sorry, I'm not exactly a Python/Flask expert, but what exactly does this PR accomplish?

File: app.py [hunk 1]
```diff
@@ -86,3 +86,7 @@ def delete_todo_by_id(id):
             del todos[i]
             return
     raise Exception("Todo not found.")
+
+
+if __name__ == '__main__':
+    app.run(debug=True)
```
Comment (w3bdesign) [inline app.py:92]: Why debug=True here?

Comment (no-0-name) [reply app.py:92]: Sorry i can be incorrect but when i clone this repository and try run app `python app.py` my cmd just close the program after i added this code and it's works, it's help me to run app.

Comment (no-0-name) [reply app.py:92]: You can check another repository, and this block of code will appear there very often, so I advise adding this

Comment (no-0-name) [reply app.py:92]: Sorry i can be incorrect but when i clone this repository and try run app `python app.py` my cmd just close the program after i added this code and it's works, it's help me to run app. Debug can helps run app, maybe i'm just stupid but if i trying run app without my code it's don't work. I also wrote debug=True just to see the errors, it's not so important if you want, you can remove it

Comment (no-0-name) [reply app.py:92]: You can check another repository, and this block of code will appear there very often, so I advise adding this

**Comment by w3bdesign** [comment]
Solved in https://github.com/w3bdesign/flask-todo/commit/2778752b17a785493af1f647b87341d97a6e487c

**Pull request closed without merging**